linode/manager
# M3-5254: Improve presentation of info & fix error-surfacing bug in LinodeConfigDialog

## Description
- [x] If devices are assigned non-sequentially (e.g. `sda` and `sdb` are assigned, then no others until `sdh`), display up to the last assigned device to the user instead of having the user click "Add a Device" until reaching that slot
- [x] Surface the errors when a user assigns a volume or a disk that is already assigned on a different slot

## How to Test
Try to add or edit a Linode's configuration and assign the same disk or volume for more than one Block Device–the error should be surfaced for each.

If you edit an existing configuration, you should see up to the last assigned slot when the modal opens, i.e. if no slots after `/dev/sdd` are assigned, you should see `/dev/sda`, `/dev/sdb`, `/dev/sdc`, and `/dev/sdd`.
